### ISSUE TITLE
Promote a stable release

### ISSUE THREAD

**Issue by locnnil** [canonical/rt-tests-snap#48]
Since the `rt-tests` snap is still in the `latest/beta` channel and hasn't seen much development progress for some time, it might be a good opportunity to promote a stable release.

**Comment by benjamin051000** [comment]
I agree. What does this take? Is there documentation on how to do it?

**Comment by locnnil** [comment]
There is no strict rule or process for that.
Apart from [this doc](https://snapcraft.io/docs/explanation/how-snaps-work/channels-and-tracks/) that explains what snap channels and tracks are, it's up to the maintainer to decide when the snap is ready to be promoted and released to stable.

As you can see on [Risk-levels](https://snapcraft.io/docs/explanation/how-snaps-work/channels-and-tracks/#risk-levels) heading under the **stable** bullet:
> Releases at this risk level are as stable as they will ever get, **according to the project’s standards**.

Previously, the decision to not be promoted to stable yet was mainly due to two factors:
- We couldn't promise/compromise with an active high-standard maintenance
- There are some existing bugs, for instance, this one:
  - https://github.com/canonical/rt-tests-snap/issues/45
